src/set/set.controller.ts

```diff
@@ -1,131 +1,118 @@
+    Param,
+    Patch,
+    Post,
+    Put,
+    Query,
+    Request,
+    UseGuards,
+} from '@nestjs/common'
+import { ApiOperation, ApiTags } from '@nestjs/swagger'
+import { ObjectId } from 'mongoose'
+import { JwtUserDto } from 'src/auth/dto/jwt.dto'
+import { JwtAuthGuard } from '../auth/strategies/jwt/jwt-auth.guard'
+import { MongoIdDto } from '../shared/dto/mongoId.dto'
+import { CreateSetDto } from './dto/create-set.dto'
+import { CreateTaskDto } from './dto/create-task.dto'
+import { UpdateSetDto } from './dto/update-set.dto'
+import { UpdateTaskDto } from './dto/update-task.dto'
+import { SetService } from './set.service'
 
 @ApiTags('set')
 @Controller('set')
 export class SetController {
-  constructor(private readonly setService: SetService) { }
+    constructor(private readonly setService: SetService) {}
 
-  @Post()
-  @UseGuards(JwtAuthGuard)
-  @ApiOperation({ summary: 'Create new set' })
-  createSet(
-    @Body() createSetDto: CreateSetDto, @Request() req) {
-    return this.setService.createSet(createSetDto, req.user);
-  }
+    @Post()
+    @UseGuards(JwtAuthGuard)
+    @ApiOperation({ summary: 'Create new set' })
+    createSet(@Body() createSetDto: CreateSetDto, @Request() req) {
+        return this.setService.createSet(createSetDto, req.user)
+    }
 
-  @Get()
-  @ApiOperation({ summary: 'Get all Sets' })
-  getAllSets() {
-    return this.setService.getAllSets();
-  }
+    @Get()
+    @ApiOperation({ summary: 'Get all Sets' })
+    getAllSets() {
+        return this.setService.getAllSets()
+    }
 
-  @Get('/full')
-  @UseGuards(JwtAuthGuard)
-  @ApiOperation({ summary: 'Get all Sets' })
-  getAllSetsFull(
-    @Request() { user }: ParameterDecorator & { user: JwtUserDto }) {
-    throw new NotImplementedException('Currently disabled')
-    return //this.setService.getAllSetsFull(user);
-  }
+    @Get(':id')
+    @ApiOperation({ summary: 'Get one Set by id' })
+    getOneSet(@Param() { id }: MongoIdDto) {
+        return this.setService.getOneSet(id)
+    }
 
-  @Get(':id')
-  @ApiOperation({ summary: 'Get one Set by id' })
-  getOneSet(
-    @Param() { id }: MongoIdDto) {
-    return this.setService.getOneSet(id);
-  }
+    @Patch(':id')
+    @UseGuards(JwtAuthGuard)
+    @ApiOperation({ summary: 'Update Set by id' })
+    updateMeta(
+        @Param() { id }: MongoIdDto,
+        @Body() updateSetDto: UpdateSetDto,
+        @Request() { user }: ParameterDecorator & { user: JwtUserDto },
+    ) {
+        return this.setService.updateSetMetadata(id, updateSetDto, user)
+    }
 
-  @Get(':id/full')
-  @UseGuards(JwtAuthGuard)
-  @ApiOperation({ summary: 'Get one Set by id' })
-  getOneSetFull(
-    @Param() { id }: MongoIdDto,
-    @Request() { user }: ParameterDecorator & { user: JwtUserDto }) {
-    throw new NotImplementedException('Currently disabled')
-    return //this.setService.getOneSetFull(id, user);
-  }
+    @Delete(':id')
+    @UseGuards(JwtAuthGuard)
+    // TODO MC: Can not reprocude error
```

Comment:
is this still relevant? If not => remove

Reply:
This is an artifact from 200 years ago